httpdocs/js/config_callbacks/scripts-list-utils.js

```diff
@@ -1220,31 +1230,44 @@ $(document).ready(function() {
 
    const add_filter_categories_dropdown = () => {
 
+      const CATEGORY_COLUMN_INDEX = 1;
+      const VALUES_COLUMN_INDEX = 3;
+
       const $dropdown = $(`
          <div class='dropdown d-inline'>
             <button class='btn btn-link dropdown-toggle' data-toggle='dropdown' type='button'>
-               <i class='fas fa-filter'></i> <span>${i18n.filter_categories}</span>
+               <span>${i18n.filter_categories}</span>
             </button>
             <div id='category-filter' class='dropdown-menu'>
             </div>
          </div>
       `);
-
+ 
       $dropdown.find('#category-filter').append(
 
          scripts_categories.map(c => {
             
+            // list element to append inside the dropdown selector
             const $list_element = $(`<li class='dropdown-item pointer'>${c}</li>`);
+            
+            // when a user click the filter category then the datatable
+            // will be filtered
             $list_element.click(function() {
 
-               if (c == 'All') {
-                  $script_table.column(1).search('').draw();
+               if (c == i18n.all) {
```

Comment:
again, you cannot rely in a localized string for this check.

Reply:
Okok, now I use another approach

Reply:
Okay, I changed the logic. Now, each category inside `scripts_categories` has two fields:
* `label`: it is the category name (string)
* `disableFilter`: it is a boolean value that when it's true it disable category filtering